x/data/server/msg_attest.go

```diff
@@ -0,0 +1,61 @@
+package server
+
+import (
+	"context"
+
+	"google.golang.org/protobuf/types/known/timestamppb"
+
+	cosmossdk "github.com/cosmos/cosmos-sdk/types"
+
+	api "github.com/regen-network/regen-ledger/api/regen/data/v1"
+	sdk "github.com/regen-network/regen-ledger/types"
+	"github.com/regen-network/regen-ledger/x/data"
+)
+
+// Attest allows for digital signing of an arbitrary piece of data on the blockchain.
+func (s serverImpl) Attest(ctx context.Context, request *data.MsgAttest) (*data.MsgAttestResponse, error) {
```

Comment:
not related to this PR, but is it correct that we have `attestors []string` and only 1 hash? i feel like the opposite would be more common, 1 attestor and multiple hashes? 🤷🏻

Reply:
That's a good point. Opened #961.